sdk/identity/azure-identity/test/perf/inc/azure/identity/test/environment_credential_test.hpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @file
+ * @brief Test the overhead of authenticating with secret credential.
+ *
+ */
+
+#pragma once
+
+#include <azure/identity.hpp>
+#include <azure/perf.hpp>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace Azure { namespace Identity { namespace Test {
+
+  /**
+   * @brief A test to measure the authentication token performance.
+   *
+   */
+  class EnvironmentCredentialTest : public Azure::Perf::PerfTest {
+  private:
+    Core::Credentials::TokenRequestContext m_tokenRequestContext;
+    std::unique_ptr<Azure::Core::Credentials::TokenCredential> m_credential;
+
+  public:
+    /**
+     * @brief Setup the test
+     *
+     */
+    void Setup() override
+    {
+      m_tokenRequestContext.Scopes.push_back(m_options.GetMandatoryOption<std::string>("Scope"));
+      if (!m_options.GetOptionOrDefault<bool>("Cache", false))
+      {
+        m_tokenRequestContext.MinimumExpiration = std::chrono::hours(1000000);
+      }
```

Comment:
What's the intent here? Why do we want to set this `MinimumExpiration`?

Reply:
to not use the credentials cache